tests/certification/pubsub/azure/servicebus/servicebus_test.go

```diff
@@ -0,0 +1,346 @@
+		Step("verify if app2 has recevied messages published to topic1", assertMessages(10*time.Second, consumerGroup2)).
+		Step("reset", flow.Reset(consumerGroup1, consumerGroup2)).
+
+		// Test : multiple publisher with different partitionkey, multiple subscriber with same consumer ID
+		// Run subscriberApplication app3
+		Step(app.Run(appID3, fmt.Sprintf(":%d", appPort+portOffset*2),
+			subscriberApplication(appID3, topicActiveName, consumerGroup2))).
+
+		// Run the Dapr sidecar with the component 3.
+		Step(sidecar.Run(sidecarName3,
+			embedded.WithComponentsPath("./components/consumer3"),
+			embedded.WithAppProtocol(runtime.HTTPProtocol, appPort+portOffset*2),
+			embedded.WithDaprGRPCPort(runtime.DefaultDaprAPIGRPCPort+portOffset*2),
+			embedded.WithDaprHTTPPort(runtime.DefaultDaprHTTPPort+portOffset*2),
+			embedded.WithProfilePort(runtime.DefaultProfilePort+portOffset*2),
+			runtime.WithSecretStores(secretStoreComponent),
+			runtime.WithPubSubs(component))).
+
+		// publish message in topic1 from two publisher apps, however there are two subscriber apps (app2,app3) with same consumerID
+		Step("publish messages to topic1", publishMessages(metadata, sidecarName1, topicActiveName, defaultTopicCreation, consumerGroup2)).
+		Step("publish messages to topic1", publishMessages(metadata1, sidecarName2, topicActiveName, defaultTopicCreation, consumerGroup2)).
+		Step("verify if app2, app3 together have recevied messages published to topic1", assertMessages(10*time.Second, consumerGroup2)).
+
+		// Test : Entitymanagement , Test partition key, in order processing with single publisher/subscriber
+		// Run subscriberApplication app4
+		Step(app.Run(appID4, fmt.Sprintf(":%d", appPort+portOffset*3),
+			subscriberApplication(appID4, topicToBeCreated, consumerGroup3))).
+
+		// Run the Dapr sidecar with the component entitymanagement
+		Step(sidecar.Run(sidecarName4,
+			embedded.WithComponentsPath("./components/consumer4"),
+			embedded.WithAppProtocol(runtime.HTTPProtocol, appPort+portOffset*3),
+			embedded.WithDaprGRPCPort(runtime.DefaultDaprAPIGRPCPort+portOffset*3),
+			embedded.WithDaprHTTPPort(runtime.DefaultDaprHTTPPort+portOffset*3),
+			embedded.WithProfilePort(runtime.DefaultProfilePort+portOffset*3),
+			runtime.WithSecretStores(secretStoreComponent),
+			runtime.WithPubSubs(component))).
+		Step(fmt.Sprintf("publish messages to topicToBeCreated: %s", topicToBeCreated), publishMessages(metadata, sidecarName4, topicToBeCreated, defaultTopicCreation, consumerGroup3)).
+		Step("interrupt network", network.InterruptNetwork(time.Minute, []string{}, []string{}, "5671", "5672")).
+		Step("wait", flow.Sleep(30*time.Second)).
+		Step("verify if app4 has recevied messages published to newly created topic", assertMessages(10*time.Second, consumerGroup3)).
+
+		// Test : disableEntityManagement optional attribute , Test processing with all optional attributes and single publisher/subscriber
```

Comment:
So this is only testing that by default entity management is enabled. That should be made more clear.